app/src/main/java/com/github/se/project/ui/profile/ProfileInfo.kt

```diff
@@ -266,27 +285,43 @@ fun ProfileInfoScreen(
                                         modifier = Modifier.testTag("priceText"))
                                   }
                             }
-
-                            Divider(color = MaterialTheme.colorScheme.outlineVariant)
-                            // Lessons Count
-                            Text(
-                                text =
-                                    "${completedLessons.size} ${if (isTutor) stringResource(id = R.string.lessons_given) else stringResource(id = R.string.lessons_taken)}",
-                                style = MaterialTheme.typography.bodyMedium,
-                                modifier = Modifier.testTag("lessonsCount"),
-                                color = MaterialTheme.colorScheme.onBackground)
-
-                            // Display Completed Lessons Section
-                            ExpandableLessonSection(
-                                section = completedLessonsSection,
-                                lessons = completedLessons,
-                                isTutor = isTutor,
-                                onClick = { lesson ->
-                                  lessonViewModel.selectLesson(lesson)
-                                  navigationActions.navigateTo(Screen.COMPLETED_LESSON)
+                            // Log Out Button at the bottom
```

Comment:
Good implementation of the sign-out feature! However, it might be worth double-checking the UI rendering on the Student side. It seems a bit broken, which could impact the user experience.